### ISSUE TITLE
Exception: Could not find DESCRIPTION_CONTAINER

### ISSUE THREAD

**Issue by Banjoei** [7x11x13/songs-to-youtube#19]
Log:
```
[16:16:56] [ERROR] Traceback (most recent call last):
  File "youtube_uploader_selenium\__init__.py", line 159, in upload_all
  File "youtube_uploader_selenium\__init__.py", line 190, in upload
  File "youtube_uploader_selenium\__init__.py", line 314, in __upload
  File "youtube_uploader_selenium\__init__.py", line 275, in __find
Exception: Could not find DESCRIPTION_CONTAINER
```

**Comment by binkelderg** [comment]
same issue here since today

**Comment by 7x11x13** [comment]
Fixed in v0.11.16

**Comment by qyZMVYcw1VA6** [comment]
having this issue now as well, v0.11.19 windows
```
[22:24:41] [ERROR] Traceback (most recent call last):
  File "youtube_uploader_selenium\__init__.py", line 159, in upload_all
  File "youtube_uploader_selenium\__init__.py", line 200, in upload
  File "youtube_uploader_selenium\__init__.py", line 324, in __upload
  File "youtube_uploader_selenium\__init__.py", line 285, in __find
Exception: Could not find DESCRIPTION_CONTAINER
```

*did try to generate a new cookies json file, no difference
